sdk/netapp/Azure.ResourceManager.NetApp/src/Generated/Models/AccountEncryption.cs

```diff
@@ -8,21 +8,29 @@
 namespace Azure.ResourceManager.NetApp.Models
 {
     /// <summary> Encryption settings. </summary>
-    internal partial class AccountEncryption
+    public partial class AccountEncryption
```

Comment:
`AccountEncryption` is too generic which may cause name collision with other Azure SDK, consider to prepend the RP name to `NetAppAccountEncryption`